<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_1</class>
 <widget class="QWidget" name="Page_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>138</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="RegisterDisplay" name="widget_Voltage" native="true">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>21</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QProgressBar" name="progressBar_Ounces">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>10</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar{
border: 0px solid black;
padding: 0px;
border-radius: 0px;
background: black;
}
QProgressBar::chunk{
background: red;
border-radius: 0px;
}</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Ounces</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="RegisterDisplay" name="widget_Ounces" native="true">
             <property name="minimumSize">
              <size>
               <width>81</width>
               <height>21</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Threshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="RegisterDisplay" name="widget_Threshold" native="true">
             <property name="minimumSize">
              <size>
               <width>81</width>
               <height>21</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="CoilDisplay" name="widget_Touching" native="true">
             <property name="minimumSize">
              <size>
               <width>81</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="RegisterDisplay" name="widget_Max" native="true">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>21</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CoilButton" name="widget_Reset" native="true">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>79</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>132</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegisterDisplay</class>
   <extends>QWidget</extends>
   <header>registerdisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CoilDisplay</class>
   <extends>QWidget</extends>
   <header>coildisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CoilButton</class>
   <extends>QWidget</extends>
   <header>coilbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
